projects/lib/src/entities/login-provider.ts

```diff
@@ -2,7 +2,7 @@ import { SocialUser } from './social-user';
 
 export interface LoginProvider {
   initialize(): Promise<void>;
-	getLoginStatus(): Promise<SocialUser>;
+	getLoginStatus(refreshAuthToken?: boolean): Promise<SocialUser>;
```

Comment:
Could you please make it an options object (similar to `signInOptions` in the `signIn` method)?

Reply:
I applied similar approach (please check it in the new commit).